.circleci/config.yml

```diff
@@ -1,16 +1,13 @@
-version: 2
-jobs:
-  build:
-    docker:
-      - image: circleci/ruby:2.4.1-node-browsers
-
-    steps:
-      - checkout
-      
-      - run:
-        name: Greeting
-        command: echo Hello, world.
-        
-      - run:
-        name: Print the Current Time
-        command: date
+# Use the latest 2.1 version of CircleCI pipeline process engine. See: https://circleci.com/docs/2.0/configuration-reference
+version: 2.1
+# Use a package of configuration called an orb.
+orbs:
+  # Declare a dependency on the welcome-orb
```

Comment:
- [x] Finish my changes
- [ ] Push my commits to GitHub
- [ ] Open a pull request